### PR TITLE
Add options to merge groups of labels

### PR DESCRIPTION
**Pull request by frheault**
Facilitate combining labels while grouping

### CODE REVIEW

**Comment by scil-admin** [comment]
Build passed ! Good Job :beers: !

**Comment by scil-admin** [comment]
Build passed ! Good Job :beers: !

**Comment by scil-admin** [comment]
Build passed ! Good Job :beers: !

File: scripts/scil_combine_labels.py [hunk 1]
```diff
@@ -84,7 +85,12 @@ def main():
             indices_per_volume.append(np.asarray(v_args[1:], dtype=int))
 
     if used_indices_all and args.out_labels_ids:
-        logging.error("'all' indices cannot be used with 'out_labels_ids'")
+        parser.error("'all' indices cannot be used with --out_labels_ids.")
+
+    if args.merge_groups and args.group_in_m or \
+            args.merge_groups and args.unique:
```
Comment (arnaudbore) [inline scripts/scil_combine_labels.py:91]: Do we need parenthesis here ?

Comment (arnaudbore) [reply scripts/scil_combine_labels.py:91]: args.merge_groups and (args.group_in_m or args.unique): should be enough here

**Review by arnaudbore — APPROVED** [review]
Just one comment otherwise LGTM

**Comment by scil-admin** [comment]
Build passed ! Good Job :beers: !

**Comment by scil-admin** [comment]
Build passed ! Good Job :beers: !

**Comment by scil-admin** [comment]
Build passed ! Good Job :beers: !

**Review by arnaudbore — APPROVED** [review]

**Pull request merged**